src/components/components/Sidebar.js

```diff
@@ -35,13 +35,20 @@ export default class Sidebar extends React.Component {
     Events.emit('selectedEntityComponentChanged', event.detail);
   }
 
+  componentCreated = (event) => {
+    Events.emit('selectedEntityComponentCreated', event.detail);
```

Comment:
Yep, I've opened an issue for that https://github.com/aframevr/aframe-inspector/issues/357